src/libraries/System.Transactions/src/System.Transactions.csproj

```diff
@@ -2,17 +2,21 @@
   <PropertyGroup>
```

Comment:
The library is being renamed from System.Transactions.Local to System.Transactions?  I don't think that's viable, both for breaking change reasons and for conflict with .NET Framework reasons.  @ericstj?

Reply:
Am happy to change it back if it's problematic - was just assuming it made sense with it now supporting distributed transactions.

Reply:
I definitely don't think it's worth going into too much trouble here. Unless this is easy to do or someone feels strongly about it, I can just drop it and keep the System.Transactions.Local name.

Reply:
(thanks for weighing in regardless!)